pkg/agent/baker_test.go

```diff
@@ -1328,6 +1328,19 @@ oom_score = 0
 					},
 				}
 			}, nil),
+		Entry("AKSUbuntu2204 IMDSRestriction with enable restriction and insert to mangle table", "AKSUbuntu2204+IMDSRestrictionOnWithMangleTable", "1.24.2",
+			func(config *datamodel.NodeBootstrappingConfiguration) {
+				config.EnableIMDSRestriction = true
+				config.InsertIMDSRestrictionRuleToMangleTable = true
```

Comment:
can we add assertions on the output here? e.g. contents of the new vars added to cse_cmd.sh to make sure they're getting the correct corresponding values?

see up above for examples

Reply:
thanks for reminding! added validation.